debug_toolbar/static/debug_toolbar/css/toolbar.css

```diff
@@ -654,3 +720,17 @@
 .djdt-hidden {
     display: none;
 }
+
+#djToggleThemeButton {
+    display: flex !important;
+    align-items: center;
+    cursor: pointer;
+}
+#djToggleThemeButton > svg {
+    margin-left: auto;
+}
+#djDebug[data-theme="light"] #djToggleThemeButton svg.theme-light,
+#djDebug[data-theme="dark"] #djToggleThemeButton svg.theme-dark,
+#djDebug[data-theme="auto"] #djToggleThemeButton svg.theme-auto {
+    display: block !important;
```

Comment:
Can you explain why the `!Important` is necessary here and above? We have tried avoiding `!important` in our CSS until now.

Reply:
Nice catch, I missed these on the final cleanup, in [e8c6820](https://github.com/jazzband/django-debug-toolbar/pull/1913/commits/e8c68200b107f0a564bdaaf29ade5028f900a84a) they are removed